Lab#1.ipynb

```diff
@@ -0,0 +1,283 @@
QRpwCAAAAoI04BQAAAEAbcQoAAACANuIUAAAAAG3EKQAAAADaiFMAAAAAtBGnAAAAAGgjTgEAAADQRpwCAAAAoI04BQAAAEAbcQoAAACANuIUAAAAAG3EKQAAAADaiFMAAAAAtBGnAAAAAGgjTgEAAADQRpwCAAAAoI04BQAAAEAbcQoAAACANuIUAAAAAG3EKQAAAADaiFMAAAAAtBGnAAAAAGgjTgEAAADQRpwCAAAAoI04BQAAAEAbcQoAAACANuIUAAAAAG3EKQAAAADaiFMAAAAAtBGnAAAAAGgjTgEAAADQRpwCAAAAoI04BQAAAEAbcQoAAACANuIUAAD/v717D9atrO8D/v3BqdzOgRjAghcEnakoDCKeVIuZ1CuT1Fu8JE5LO42NoqbVGIXGYKeoscKIlkGtePkjjol2RmO8m2okxsyoaFBxRgzUGFEjSoBa4AgHL/z6x7u2bE/33ufd55y9n332+Xxm1qzb8zz7d2ZY7P1+37WeBQAwjHAKAAAAgGGEUwAAAAAMI5wCAAAAYBjhFAAAAADDCKcAAAAAGEY4BQAAAMAwwikAAAAAhhFOAQAAADCMcAoAAABgg9ixI7ngguTYY5ODDpqtL7hgdnyz2jK6AAAAAABmAdQjH5l84xvJzp2zYzfdlLz2tcn73pdccUWydevYGteCO6cAAAAANoCLL/75YGrBzp2z4xdfPKautSacAgAAANgA3vzm/z+YWrBzZ3LZZetbz3oRTgEAAABsADffvHfn91fCKQAAAIAN4Oij9+78/ko4BQAAALAB/M7vJIceuvS5Qw9NXvCC9a1nvaw6nKqq46rq0qr6RlXtrKobqurDVfW4vSmkqo6sqldX1d9W1e1VdXNVXV5Vz9xNv+uqqneznLs3tQEAAACstfPOSx74wGTLlp8/vmXL7Ph5542pa62tKpyqqtOSfDXJi5I8IMmdSY5J8qQkf1FVL9uTIqrqvkmuSvLyJCcn+WmSI5M8Nsl7q2qeKb9+kOSGZZYf7kldAAAAAOtl69bkiiuSM89MDj88qZqtzzxzdnzr1tEVro0tu28yU1WHJflQkqOTfDnJv+vuq6vqyCT/NclLk1xYVV/q7k+sYtxK8qdJTkpyXZKzu/uzVXVokhcmuSjJ86dx377CUE/v7r+a9+cCAAAAbDRbtyaPecxs2fX4ZrWaO6eel+T+SXYkeXJ3X50k3X1rd5+b5ANTuwtXWcNTkzwiyV1Jntbdn53G3dndFyd5w9TuVVV1j1WODQAAAMAGtppw6uxp/e7u/u4S5y+e1mdU1cl7MO4nu/uqJc6/LkknOS6zx/wAAAAA2CTmCqeqaluSh0+7H1+m2RVJbpm2VxMiPXqlcacg7Oo9GBcAAACADW7eO6cenKSm7auXatDddyW5dtp9yDyDVtW9MptQfdlxJ1+bY9xLqurGqvpRVX2/qj5WVf+mqg6epxYAAAAA1t+84dTxi7avX6HdwrnjV2izVuOenuTwJDuT/NMkv5bkXUkur6pfmLMeAAAAANbRvOHUEYu271ih3e3Tet455PfFuB9I8owkx3T3Ed19ZGYTt78us0nW/2WS96xURFWdU1VXVtWVN95445ylAwAAALC35g2navdN9sjicXtPBujuF3f3n3X3zYuOfbu7z0vywunQE6rqrBXGeFt3b+/u7ccee+yelAEAAADAHpg3nNqxaPuwFdodvkT7ecc9fNlWqx93wWVJrpu2n7zKvgAAAACssXnDqcXzQd17hXYL5743eNwkSXd3kr+Zdh+wmr4AAAAArL15w6lrcvdjd6cs1aCqDkryoGn3a0u12VV335jkppXGnSy8pW+ucXctbeHH7UFfAAAAANbQXOFUd9+W5Mpp9wnLNHtEkqOm7ctXUcOnVhq3qu6Tu4Or1Yybqqok26fd61bTFwAAAIC1N++dU0ny7ml9dlUdv8T5c6f1F7v72j0Y96yqeugS51+S2d1P38vdQVaSn4VPK3lekhOn7Y+uoiYAAAAA1sFqwqm3JvlWkm1JPlJVD0mSqtpWVa9N8vSp3fm7dqyqnpZXLDHuB5N8fqrl/VX1yKnPIVX10iQvntpd0N0/2qXvG6rq0qr65ar62UTtVXW/qrooyZumQ5/q7j9fxb8VAAAAgHWwZd6G3X1HVT01s0frzkhydVXdmmRrZsFSJzm/uz+xmgK6u6vqmUn+OslJST5XVTuSHLqovrd099uX6L4tyb9P8qIkd1XVLUkOTnLkojafTvLM1dQEAAAAwPqYO5xKku7+SlWdmuQPkjwpyX2S3JzkC0ku6e5VzQm1aNx/qKrTk/x+ZndgnZjktiRXJbmsu9+7TNe3JLkxyZlJTkhydGZB2XcymyPrfyZ5X3fftSd1AQAAALC2VhVOJUl3fz/J707LvH12NzdUuvvWJC+flnnHvSLJFfO2BwAAAGBjWc2cUwAAAACwTwmnAAAAABhGOAUAAADAMMIpAAAAAIYRTgEAAAAwjHAKAAAAgGGEUwAAAAAMI5wCAAAAYBjhFAAAAADDCKcAAAAAGEY4BQAAAMAwwikAAAAAhhFOAQAAADCMcAoAAACAYYRTAAAAAAwjnAIAAABgGOEUAAAAAMMIpwAAAAAYRjgFAAAAwDDCKQAAAACGEU4BAAAAMIxwCgAAAIBhhFMAAAAADCOcAgAAAGAY4RQAAAAAwwinAAAAABhGOAUAAADAMMIpAAAAAIYRTgEAAAAwjHAKAAAAgGGEUwAAAAAMI5wCAAAAYBjhFAAAAADDCKcAAAAAGEY4BQAAAMAwwikAAAAAhhFOAQAAADCMcAoAAACAYYRTAAAAAAwjnAIAAABgGOEUAAAAAMMIpwAAAAAYRjgFAAAAwDDCKQAAAACGEU4BAAAAMIxwCgAAAIBhhFMAAAAADCOcAgAAAGAY4RQAAAAAwwinAAAAABhGOAUAAADAMMIpAAAAAIYRTgEAAAAwjHAKAAAAgGGEUwAAAAAMI5wCAAAAYBjhFAAAAAD
```

Comment:
Maybe break up your notebook into sections so that the reader knows when you have moved on to a new idea.